pkg/util/test/histogram.go

```diff
@@ -36,7 +36,7 @@ func GenerateTestGaugeFloatHistogram(i int) *histogram.FloatHistogram {
 // explicit decoded version of GenerateTestHistogram and GenerateTestFloatHistogram
 func GenerateTestSampleHistogram(i int) *model.SampleHistogram {
 	return &model.SampleHistogram{
-		Count: model.FloatString(10 + i*8),
+		Count: model.FloatString(12 + i*9),
```

Comment:
this was necessary due to this change https://github.com/grafana/mimir-prometheus/commit/aa82fe198f66f4e2a41647fd0845834acf365e0e#diff-4c7063fbe57f049434ef7ed4d6360beb38a6cb4bd8154b2d5a2143557a5f52e3